src/realm/sync/network/network.hpp

```diff
@@ -321,8 +322,10 @@ class Service {
     ///
     /// Register the sepcified completion handler for immediate asynchronous
     /// execution. The specified handler will be executed by an expression on
-    /// the form `handler()`. If the the handler object is movable, it will
-    /// never be copied. Otherwise, it will be copied as necessary.
+    /// the form `handler(status)` where status is a Status object whose value
+    /// will always be OK, but may change in the future. If the handler object
```

Comment:
The handler is not copyable it's a UniqueFunction.

Reply:
This is a generic comment, since `handler` is a template type `H` which could potentially be copyable. Our use of it uses the UniqueFunction, which is not copyable, like you mentioned.